### PR TITLE
Fixing node finalize for health map validation

### PR DESCRIPTION
**Pull request by sradhanand-pati**
Signed-off-by: sradhanandpati <sradhanand.pati@seagate.com>

### CODE REVIEW

File: lr-cli/cortx_setup/commands/node/finalize.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def _validate_health_map(self):
         self.logger.debug("Validating node health check")
         resource_paths = [HEALTH_PATH, MANIFEST_PATH]
         for resource in resource_paths:
-            path = get_pillar_data(resource)
+            path = get_pillar_data(resource).split('://')[1]
```
Comment (mazinamdar) [inline lr-cli/cortx_setup/commands/node/finalize.py:41]: can  you add this split after below check, if path is present then only do it.

File: lr-cli/cortx_setup/commands/node/finalize.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def _validate_health_map(self):
         self.logger.debug("Validating node health check")
         resource_paths = [HEALTH_PATH, MANIFEST_PATH]
```
Comment (mazinamdar) [inline lr-cli/cortx_setup/commands/node/finalize.py:39]: can you remove MANIFEST_PATH as we don't have manifest in place

Comment (sradhanand-pati) [reply lr-cli/cortx_setup/commands/node/finalize.py]: Okay.

File: lr-cli/cortx_setup/commands/node/finalize.py (hunk 1 above)
Comment (sradhanand-pati) [reply lr-cli/cortx_setup/commands/node/finalize.py]: Yes. i have moved the split after the path check..

**Review by mazinamdar — APPROVED** [review]
looks good to me

**Review by 83bhp — CHANGES_REQUESTED** [review]
Minor comment. Otherwise looks good.

**Review by 83bhp — APPROVED** [review]

**Pull request merged**